src/unpkg.mjs

```diff
@@ -27,7 +27,7 @@ async function transformBody(resp, req) {
   const respURL = new URL(resp.url);
   if (resp.ok
     && resp.status === 200
-    && url.pathname.indexOf('@adobe/helix-rum-js')) {
+    && url.pathname.indexOf('@adobe/helix-rum-js') > 0) {
```

Comment:
@trieloff I added `> 0` as I think the fact that this wasn't there was a bug. `String.indexOf()` returns -1 if the string isn't there and `true && -1` is still `true`.